app/configurator/components/dataset-search.tsx

```diff
@@ -383,33 +384,135 @@ export const SearchDatasetBox = ({
   );
 };
 
+const defaultNavItemTheme = {
+  activeTextColor: "white",
+  activeBg: "primary",
+  textColor: "initial",
+  countColor: "monochrome800",
+  countBg: "monochrome300",
+};
+
+const themeNavItemTheme = {
+  activeBg: "success",
+  activeTextColor: "white",
+  textColor: "initial",
+  countColor: "success",
+  countBg: "successLight",
+};
```

Comment:
I would prefer to have categoryDark categoryLight instead of success / successLight

Reply:
+1

Reply:
Done